src/Alestra.ts

```diff
@@ -2,15 +2,13 @@ import 'module-alias/register';
 import 'reflect-metadata';
 import { AlestraClient } from '@lib/AlestraClient';
 import { CLIENT_OPTIONS, TOKEN } from '@root/config';
-import { KlasaClientOptions } from 'klasa';
 import { inspect } from 'util';
-inspect.defaultOptions.depth = 1;
-
-AlestraClient.defaultGuildSchema.add('supportChannels', 'TextChannel', { array: true });
+import * as colorette from 'colorette';
 
-const client = new AlestraClient(CLIENT_OPTIONS as KlasaClientOptions);
-client.token = TOKEN;
+inspect.defaultOptions.depth = 1;
+colorette.options.enabled = true;
 
-client.connect().catch((error) => {
-	client.console.error(error);
```

Comment:
```
client.token = TOKEN

client.connect()
```

This was still the dumbest and ao-est decision of Klasa Core ever if you ask me. Thank god we don't have that anymore.